src/examples/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) Contributors to the OpenEXR Project.
 
+cmake_minimum_required(VERSION 3.12)
+
+project(OpenEXRExamples)
+
+find_package (OpenEXR)
```

Comment:
maybe tuck in the space before paren for consistency with the rest of the file?

Reply:
Done.